### PR TITLE
vcpkg 2022-07-21

### PR DESCRIPTION
**Pull request by owine**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/vcpkg.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Vcpkg < Formula
   desc "C++ Library Manager"
   homepage "https://github.com/microsoft/vcpkg"
-  url "https://github.com/microsoft/vcpkg-tool/archive/2022-07-14.tar.gz"
-  version "2022.07.14"
-  sha256 "e7f4783d0c30c074029a08cf83443838dd3cee1610458ce2fc0fa9a8f7f4411d"
+  url "https://github.com/microsoft/vcpkg-tool/archive/2022-07-21.tar.gz"
+  version "2022-07-21"
```
Comment (carlocab) [inline Formula/vcpkg.rb:5]: Any reason we're changing from `.`s to `-`s?

Comment (owine) [reply Formula/vcpkg.rb:5]: Just matching upstream and livecheck

Comment (carlocab) [reply Formula/vcpkg.rb:5]: Last I checked their actual `vcpkg` build, I think `--version` used `.`s though. Did they change that (or am I misremembering)?

Comment (owine) [reply Formula/vcpkg.rb:5]:
```
  ~ vcpkg --version
vcpkg package management program version 2022-07-14-2022.07.14
```

Comment (SMillerDev) [reply Formula/vcpkg.rb:5]: That's not confusing at all 😓

Comment (iMichka) [reply Formula/vcpkg.rb:5]: Other package managers kept the dots: https://repology.org/project/vcpkg/versions.
I would rather keep them too.

**Review by iMichka — APPROVED** [review]
Thanks!

**Pull request closed without merging**